src/components/main/Item.tsx

```diff
@@ -61,16 +60,17 @@ const Item: FC<Props> = ({ item, memberships }) => {
     <Thumbnail
       id={item.id}
       thumbnailSrc={
-        getEmbeddedLinkExtra(extra as EmbeddedLinkItemExtra)?.thumbnails[0]
+        item.type === ItemType.LINK
+          ? getEmbeddedLinkExtra(item.extra)?.thumbnails?.first()
+          : undefined
```

Comment:
Do the apps also have thumbnails? Also, what if we want to support more types in the future? Should it be checked generically for the specific extra field?

Reply:
Mhmmm no idea, I just translated the same as in the js file.